### ISSUE TITLE
Tasks imported from a package created with the fractal-tasks-template don't include information from meta file

### ISSUE THREAD

**Issue by adrtsc** [fractal-analytics-platform/fractal-tasks-template#2]
Hi,
I tried to make a package from a few tasks I've created using the fractal-tasks-template. Everything went fine up until running the tasks that I collected from the package in fractal. The issue I run into is that the template does not specify any of the information usually included in the task meta-file. 

Also, when tasks are collected from a package, the user that imported them is not the owner of the task, meaning that they can't be edited (and therefore I can't add a meta-file via the fractal client).

Did I maybe miss some step in the template creation (I tried to adhere to the instructions, but maybe something slipped by me or was not well documented)? 

Thanks!

**Comment by tcompa** [comment]
Thanks for reporting!

Did you add a `meta` property to the task in the manifest? This should be done by hand, as there is no obvious way for the template to guess what to put there.

This means e.g. modifying the manifest as in
```json
{
  "manifest_version": "1",
  "task_list": [
    {
      "name": "Thresholding Task",
      "executable": "thresholding_task.py",
      "input_type": "zarr",
      "output_type": "zarr",
      "meta": {                                                  # START OF NEW BLOCK
        "cpus_per_task": 4,
        "mem": 16000,
        "parallelization_level": "image"
      },                                                         # END OF NEW BLOCK
      "args_schema": {
        "title": "ThresholdingTask",
        "type": "object",
        "properties": {
          "input_paths": {
            "title": "Input Paths",
            "type": "array",
            "items": {
              "type": "string"
            },
            "description": "Path to the parent folder of the NGFF image. This task only supports a single input path. (standard argument for Fractal tasks, managed by Fractal server)."
          },
          "output_path": {
            "title": "Output Path",
            "type": "string",
            "description": "This argument is not used in this task. (standard argument for Fractal tasks, managed by Fractal server)."
          },
          "component": {
            "title": "Component",
            "type": "string",
            "description": "Path of the NGFF image, relative to `input_paths[0]`. (standard argument for Fractal tasks, managed by Fractal server)."
          },
          "metadata": {
            "title": "Metadata",
            "type": "object",
            "description": "This argument is not used in this task. (standard argument for Fractal tasks, managed by Fractal server)."
          }
        },
        "required": [
          "input_paths",
          "output_path",
          "component",
          "metadata"
        ],
        "additionalProperties": false
      },
      "docs_info": "Short description of thresholding_task.\n\nLong description of thresholding_task.",
      "docs_link": "https://example.com"
    }
  ],
  "has_args_schemas": true,
  "args_schema_version": "pydantic_v1"
}
```

@adrtsc does this solve your issue?

In terms of documentation, here are some possible improvement
1. Explicitly mention that this step is to be done by-hand, in the instructions (this is useful).
2. Maybe also include an empty `meta` property in the default manifest, so that the user can guess where new properties should go.

@adrtsc do you see these changes as sufficiently helpful?

**Comment by tcompa** [comment]
> Also, when tasks are collected from a package, the user that imported them is not the owner of the task, meaning that they can't be edited (and therefore I can't add a meta-file via the fractal client).

This is a known current behavior; maybe it will change in the future (discussion started at https://github.com/fractal-analytics-platform/fractal-server/issues/1084).

**Comment by adrtsc** [comment]
Hi @tcompa, I see, indeed I did not put the meta property in my manifest manually. I think your suggestions would be perfectly suitable to keep people from running into the same thing in the future.

Thanks!

**Comment by tcompa** [comment]
Instructions fixed as of version v0.1.2 of this template.
Note that the version is visible when you run copier:
```
$ copier copy gh:fractal-analytics-platform/fractal-tasks-template your-package-name
🎤 Your full name (used in the project metadata)
   Your Name
🎤 Your email address (used in the project metadata)
   first.last@example.com
🎤 Name of your project (prefer hyphens to underscores)
   my-project
🎤 Name of your top-level Python package (the one appearing in Python `import` statements)
   my_project
🎤 A URL for your project home page or documentation (optional)
 
🎤 A short description of the package
   Package description.

Copying from template version 0.1.2                    # <------------ see here
 identical  .
 identical  LICENSE
 identical  pyproject.toml
 identical  src
 identical  src/my_project
...
```

(I'm mentioning it in view of future changes, even if in this case nothing really changed apart from the README)

**Comment by jluethi** [comment]
Great, thanks for updating the documentation for this @tcompa ! And thanks for reporting it @adrtsc !
